examples/inabox-tag-integration.js

```diff
@@ -30,9 +30,15 @@ Array.prototype.push.apply(
  * A sample bootstrap script that is to be run in ads tag.
  *
  * @param {!Window} win
+ * @param {boolean|undefined} compiled Whether to use the compiled host script
  */
-function initInaboxHost(win) {
-  const hostScriptUrl = 'http://localhost:8000/dist/amp-inabox-host.js';
+function initInaboxHost(win, compiled) {
+  let hostScriptUrl;
+  if (compiled) {
```

Comment:
this should not be needed . you can run against compiled code by

```
gulp test --compiled --files=test/integration/test-amp-inabox.js
```

Reply:
The amp-inabox-host.js file is not minimized, and I don't see it being so even when I test with your command. The inabox-tag-integration.js script is not part of the build system, so I have to explicitly tell it to fetch the compiled version with that change.

Reply:
I see the problem. A better place to handle this logic is the node server side `app.js`, as we normally decide serving mode at server side not client side.

Reply:
The change's almost ready to commit. One small question though - the script fetches the host script from port 8000, while the test server's on 8081. It looked like it still worked on Travis from your previous PRs that added inabox tests, so I assume that gulp serve is being run on Travis somehow?

Reply:
That's a very good question. Didn't think about that before. @rsimha do you have the answer why it is working?

Reply:
We do start a server on Travis while running tests.

https://github.com/ampproject/amphtml/blob/784ca1d5740fe627dec89dec703e78965e330855/build-system/tasks/runtime-test/index.js#L340-L347

Reply:
Does it also run on port 8000 also? My initial question was that the previous tests were getting the host script from that port and somehow still working. Thanks.

Reply:
I believe there is code in `build-system/app.js` that runs on port 8000. @erwinmombay can confirm.